integrationtest/integration_test.go

```diff
@@ -246,20 +315,47 @@ func TestFailWithBadTokenReviewerJwt(t *testing.T) {
 func TestAuthAliasMetadataAssignment(t *testing.T) {
 	// annotate the service account
 	expMetadata := map[string]string{
-		"foo": "bar",
-		"bar": "baz",
+		"key-1": "foo",
+		"key-2": "bar",
 	}
 
-	annotationPrefix := "auth-metadata.vault.hashicorp.com/"
+	const annotationPrefix = "vault.hashicorp.com/alias-metadata-"
 	annotations := map[string]string{}
 	for k, v := range expMetadata {
 		annotations[annotationPrefix+k] = v
 	}
 	annotateServiceAccount(t, "vault", annotations)
 
-	client, cleanup := setupKubernetesAuth(t, "vault", nil, nil)
+	client, cleanup := setupKubernetesAuth(t, map[string]interface{}{
+		"kubernetes_host":                   "https://kubernetes.default.svc.cluster.local",
+		"use_annotations_as_alias_metadata": true,
+	})
 	defer cleanup()
 
+	// create policy
+	secret, err := client.Logical().Read("sys/auth/kubernetes")
+	if err != nil {
+		t.Fatalf("Expected successful auth configuration GET but got: %v", err)
+	}
+
+	mountAccessor, ok := secret.Data["accessor"]
+	if !ok {
+		t.Fatal("Expected auth configuration GET response to have \"accessor\"")
+	}
+
+	const policyNameFoo = "alias-metadata-foo"
+	const kvPath = "root"
```

Comment:
nit: root is a bit of a confusing path to use, because it's quite an overloaded term

```suggestion
	const kvPath = "kv-v1"
```